### PR TITLE
updating github templates

### PR DESCRIPTION
**Pull request by zricethezav**
### Description:
This PR updates some of the `.github` templates to make issue content a little more concise and adds a PR template. It also changes `test-forks` to `test-community` so contributors are encouraged to run `make test-community` locally -- sounds a little better than running `make test-forks` locally.

Open to suggestions on any/all of this

### Checklist:
* [x] Does your PR pass tests (`make test-community`)?
* [x] Have you lint your code locally prior to submission (`make lint` this requires [golangci-lint](https://golangci-lint.run/usage/install/#local-installation))?

### CODE REVIEW

**Review by mcastorina — APPROVED** [review]
Nothing blocking, just added a few thoughts.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -6,28 +6,26 @@ labels: enhancement, needs triage
...
-* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
-* If you are interested in working on this issue or have submitted a pull request, please leave a comment
-
-### Description
+Please Review the [Community Note](../community_note.md) before submitting
+<!-- ### Community Note -->
+<!--  -->
+<!-- * Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request -->
+<!-- * Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request -->
+<!-- * If you are interested in working on this issue or have submitted a pull request, please leave a comment -->
```
[the remark this answers is not in the document]
Comment (zricethezav) [reply .github/ISSUE_TEMPLATE/feature_request.md:15]: No, it doesn't show up. It's just for reference and to put the note in front of their eyes when submitting an issue.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,3 +2,11 @@
 Please create an issue to collect feedback prior to feature additions. Please also reference that issue in any PRs.
 If possible try to keep PRs scoped to one feature, and add tests for new features.
 -->
+
+### Description:
+Explain the purpose of the PR.
+
+### Checklist:
+* [ ] Does your PR pass tests (`make test-community`)?
+* [ ] Have you lint your code locally prior to submission (`make lint` this requires [golangci-lint](https://golangci-lint.run/usage/install/#local-installation))?
```
[the remark this answers is not in the document]
Comment (zricethezav) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: much better

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mcastorina) [reply .github/ISSUE_TEMPLATE/feature_request.md]: They don't show up when the issue is created, but the user creating the issue will still see them during creation?

[the remark this answers is not in the document]
Comment (zricethezav) [reply .github/ISSUE_TEMPLATE/feature_request.md]: It's behaves like a standard comment. Meaning when the contributor is editing an issue they will see the comment but the comment will not be rendered by markdown

[the remark this answers is not in the document]
Comment (zricethezav) [reply .github/ISSUE_TEMPLATE/feature_request.md]: In either case, I removed the comment

**Pull request merged**